lib/chef/knife/ec_key_import.rb

```diff
@@ -25,6 +25,7 @@ class Knife
     class EcKeyImport < Chef::Knife
 
       include Knife::EcKeyBase
+      include Knife::EcBase
```

Comment:
I think this is going to also pull in the common options for the `knife ec backup` and `knife ec restore` commands which might not make sense here.